justfile

```diff
@@ -1,29 +1,42 @@
+# Variables
+PYTHON_VERSION := shell("python -c \"import sys;v=f'{sys.version_info.major}.{sys.version_info.minor}';sys.stdout.write(v)\"")
```

Comment:
Issue: If the system version updates, then the virtualenv disappears without warning.

Reply:
That's also a good point; coming from Makefiles, the venv would normally just be recreated automatically (but only if it didn't already exist); here there is a need to run the setup command once. I'm not aware of a builtin mechanism for building something only if it doesn't exist.

Do you know if ``just`` supports something to that effect?